jsonrpc/web3_endpoint_test.go

```diff
@@ -61,11 +62,13 @@ func TestWeb3EndpointClientVersion(t *testing.T) {
 
 	assert.NoError(t, expectJSONResult(resp, &res))
 	assert.Contains(t, res,
-		fmt.Sprintf(
-			clientVersionTemplate,
+		fmt.Sprintf("%s-%d/%s/%s-%s/%s",
```

Comment:
Why you couldn't use `clientVersionTemplate`?


```suggestion
		fmt.Sprintf(clientVersionTemplate,
```

Reply:
Because this test should expect a specific format returning by endpoint. If `clientVersionTemplate` has a mistake, this test should fail. If we use this variable here, it won't fail in case of a mistake.